locations/spiders/porsche_north_america.py

```diff
@@ -0,0 +1,42 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.geo import city_locations
+from locations.hours import OpeningHours
+
+
+class PorscheNorthAmericaSpider(scrapy.Spider):
+    name = "porsche_north_america"
+    item_attributes = {"brand": "Porsche", "brand_wikidata": "Q40993"}
+    countries = ["us", "ca"]
```

Comment:
Looks like store locator API is central. All countries are listed in JSON blob inside https://www.porsche.com/. Maybe it makes sense to do a single spider for all countries, how do you think?